bin/add-video-chat-instances-icons.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs')
+const process = require('process')
+
+//const { VIDEO_CHAT_INSTANCES } = require('../src/shared/constants.ts')
+// TODO instead of duplicating these data
+// Videochat Server URLs and icons
+const VIDEO_CHAT_INSTANCES = {
+  systemli: {
+    name: 'Systemli',
+    url: 'https://meet.systemli.org/$ROOM',
+    icon: '../images/videochat_instances/systemli.ico',
+  },
+  autistici: {
+    name: 'Autistici',
+    url: 'https://vc.autistici.org/$ROOM',
+    icon: '../images/videochat_instances/autistici.ico',
+  },
+}
+
+const getData = async () => {
+  let data = {}
+  for (const [id, props] of Object.entries(VIDEO_CHAT_INSTANCES)) {
+    for (const i = 1; i <= 3; i++) { // max 3 retries
+      const response = await fetch(props.url.replace('$ROOM', '/images/favicon.ico'))
```

Comment:
might work, but parsing url and just taking hostname + `/favicon.ico` might be better. but ok for now as long as the default instances are jitsi 

but do not write code to parse urls yourself, just use https://nodejs.org/docs/latest-v18.x/api/url.html